datadog/resource_datadog_synthetics_test_.go

```diff
@@ -168,7 +168,13 @@ func syntheticsTestRequest() *schema.Resource {
 				Description: "Timeout in seconds for the test.",
 				Type:        schema.TypeInt,
 				Optional:    true,
-				Default:     60,
+				DiffSuppressFunc: func(key, old, new string, d *schema.ResourceData) bool {
```

Comment:
Let's remove this `DiffSuppressFunc` to avoid any unintended consequences. Having a one-time drift because of this change should not be an issue for our customers.

Reply:
I agree, and it turns out that having a "configured timeout by default" is a bug on its own: that configured timeout would override an org level default timeout.

So let's accept this one-time drift to enforce the fixing of that bug 👍

Reply:
I changed the label of the PR to be a `changelog/bugfix` instead